### PR TITLE
Adding TestVolumeClusterResizeByAddingNodes

### PR DESCRIPTION
**Pull request by ramkrsna**
I create a cluster , add nodes to the cluster and then I add devices to the node existing cluster. I populate should consume all storage, then try to create a volume and expect failure. Then add more storage. Then create a volume and use all storage, expect success. Then create another volume and expect failure.

### CODE REVIEW

File: apps/glusterfs/app_volume_test.go [hunk 1]
```diff
@@ -1034,3 +1034,90 @@ func TestVolumeClusterResizeByAddingDevices(t *testing.T) {
...
+	}
+
+	// Add devices to that perticular node
+
+	n := NewNodeEntryFromRequest(noderequest)
+	n.DeviceAdd("xyz")
+	tests.Assert(t, len(n.Devices) == 1)
+
+	// Add two devices to the cluster
+	for _, nodeId := range clusterInfo.Nodes {
```
Comment (lpabon) [inline apps/glusterfs/app_volume_test.go]: Here the tests add devices to all nodes.  What we need is for the test to add devices to the node in 1098.

File: apps/glusterfs/app_volume_test.go [hunk 2]
```diff
@@ -1034,3 +1034,90 @@ func TestVolumeClusterResizeByAddingDevices(t *testing.T) {
...
+		Hostnames: api.HostAddresses{
+			Manage:  []string{"manage"},
+			Storage: []string{"storage"},
+		},
+		Zone: 1,
+	}
+
+	// Add devices to that perticular node
+
+	n := NewNodeEntryFromRequest(noderequest)
```
Comment (lpabon) [inline apps/glusterfs/app_volume_test.go]: It would be better if you use the client api to add a new node.  It would return node information.

**Comment by lpabon** [comment]
Also, the commit message above uses the word "I" a lot.  I would suggest instead creating a commit message describing the patch itself.  You can go ahead and edit the message.

**Comment by obnoxxx** [comment]
@ramkrsna , any update on this PR?

**Comment by ramkrsna** [comment]
I'll work on a newer patchset for this PR. Will update it the #461 . Having issues with the client calls, closing this patchset .

**Pull request closed without merging**